BinariesCheck.py

```diff
@@ -300,7 +300,7 @@ def __init__(self, pkg, path, file, is_ar, is_shlib):
 bin_regex = re.compile(r'^(/usr(/X11R6)?)?/s?bin/')
 soversion_regex = re.compile(r'.*?([0-9][.0-9]*)\\.so|.*\\.so\\.([0-9][.0-9]*).*')
 reference_regex = re.compile(r'\.la$|^/usr/lib(64)?/pkgconfig/')
-usr_lib_exception_regex = re.compile(Config.getOption('UsrLibBinaryException', r'^/usr/lib(64)?/(perl|python|ruby|menu|pkgconfig|ocaml|lib[^/]+\.(so|l?a)$|bonobo/servers/)'))
+usr_lib_exception_regex = re.compile(Config.getOption('UsrLibBinaryException', r'^/usr/lib(64)?/(perl|python|ruby|menu|pkgconfig|ocaml|lib[^/]+\.(so|l?a)$|bonobo/servers/|\\.build-id)'))
```

Comment:
There's a double backslash escape here, note that the regex is a raw string nowadays. I'll push a fixed commit shortly.